### PR TITLE
NU IS AL should have no break opportunity

### PR DESCRIPTION
**Pull request by makotokato**
When running Firefox's reftest with ICU4X integration, I found ICU4X segmenter data bug. "4:before" (NU IS AL AL ...)  shouldn't have break opportunity, but data file has break opportunity rule.

### CODE REVIEW

File: components/segmenter/tests/testdata/LineBreakExtraTest.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Additional line breaking tests, not in LineBreakTest.txt
```
Comment (robertbastian) [inline components/segmenter/tests/testdata/LineBreakExtraTest.txt:1]: Are these hand-written or sourced from somewhere? Please add to the comment and link the PR for context

**Comment by robertbastian** [comment]
Run `cargo make bakeddata components/segmenter` to fix CI

**Comment by Manishearth** [comment]
I still have a hard time understanding how these toml rules map to the spec but this looks good, r=me after tests pass

File: components/segmenter/tests/testdata/LineBreakExtraTest.txt (hunk 1 above)
Comment (makotokato) [reply components/segmenter/tests/testdata/LineBreakExtraTest.txt:1]: This is hand-writing. SentenceBreak test has same way to add new test excepting to Unicoide tools'  test.

**Review by robertbastian — APPROVED** [review]

**Review by aethanyc — APPROVED** [review]

**Pull request merged**